.github/workflows/stardis-tests.yml

```diff
@@ -0,0 +1,79 @@
+name: Stardis Tests
+
+on:
+  push:
+    branches:
+    - '*'
+
+  pull_request:
+    branches:
+    - '*'
+    
+    types:
```

Comment:
I don't think we need this at the moment because we won't want to trigger these tests using labels or comments.

Reply:
workflow dispatch is nice.

Reply:
Fixed it